app/src/renderer/components/transactions/LiBankTransaction.vue

```diff
@@ -4,7 +4,17 @@
     :time="transaction.time"
     :block="transaction.height"
   >
-    <template v-if="sent">
+    <template v-if="address === ''">
+      <div slot="caption">
+        Transfer&nbsp;<b>{{ full(atoms(coins.amount)) }}</b><span>&nbsp;{{ coins.denom.toUpperCase() }}</span>
```

Comment:
```suggestion
        Sent&nbsp;<b>{{ full(atoms(coins.amount)) }}</b><span>&nbsp;{{ coins.denom.toUpperCase() }}</span>
```

Reply:
transfer or sent ? I feel used _Transfer_ bc i feel _Sent_ is more like you're sending something, whereas here is a transfer from a random address to another

Reply:
right. i see. but it's still _sent_ _from_ and _to_

Reply:
updated